creator-node/src/services/stateMachineManager/stateReconciliation/index.js

```diff
@@ -100,14 +99,6 @@ class StateReconciliationManager {
       }
     })
 
-    // Clear any old state if redis was running but the rest of the server restarted
-    if (clusterUtils.isThisWorkerInit()) {
```

Comment:
same q

Reply:
same thing - this logic gets applied in makeQueue() now